Automattic/sensei
# Add box-shadow back to the Sensei tour

Resolves #7504
Based off of https://github.com/Automattic/sensei/pull/7538

## Proposed Changes

* ~~After thinking about the proposed options in the issue, I decided to go with the one to hide metaboxes. Making something with a dynamic position doesn't seem something safe to handle without issues, and changing the position to the top I don't think would be nice.~~
* ~~I think this approach is nice like a "focus mode" when the tour is open explaining the features.~~
* It adds a box-shadow to the Sensei tour, so it's better visible with a white background.

## Testing Instructions


1. With Sensei Pro activated, open the lesson editor with the quiz block added.
2. Make sure that when the Tour is open, and you can clearly see the tour limited through a box shadow over the content drip that is white.

## Pre-Merge Checklist
